src/main/antlr4/org/reaktivity/nukleus/maven/plugin/internal/parser/Nukleus.g4

```diff
@@ -291,6 +301,10 @@ HEX_LITERAL
    : '0' ('x' | 'X') HEX_DIGIT+ INTEGER_TYPE_SUFFIX?
    ;
 
+STRING_LITERAL
+   : '"' (~["\r\n])* '"'
```

Comment:
Let's hoist this `"` character as `QUOTE`, and also rename `STRING_LITERAL` to `string_literal`.